velox/type/Timestamp.h

```diff
@@ -31,26 +31,34 @@ class time_zone;
 namespace facebook::velox {
 
 struct TimestampToStringOptions {
-  enum Precision : int8_t {
+  enum class Precision : int8_t {
     kMilliseconds = 3,
     kNanoseconds = 9,
-  } precision = kNanoseconds;
+  };
+
+  Precision precision = Precision::kNanoseconds;
 
   bool zeroPaddingYear = false;
   char dateTimeSeparator = 'T';
-  bool dateOnly = false;
-};
 
-// Our own version of gmtime_r to avoid expensive calls to __tz_convert.  This
-// might not be very significant in micro benchmark, but is causing significant
-// context switching cost in real world queries with higher concurrency (71% of
-// time is on __tz_convert for some queries).
-//
-// Return whether the epoch second can be converted to a valid std::tm.
-bool epochToUtc(int64_t seconds, std::tm& out);
+  // When TimestampToStringOptions.mode = Mode::kFull, return ISO 8601
```

Comment:
Drop TimestampToStringOptions since this comment is inside the TimestampToStringOptions struct.

Use ///